documentation/docs/configuration.md

```diff
@@ -185,33 +173,39 @@ The built-in fallback (excerpt - full version [here](https://github.com/DataSQRL
 ```jsonc
 {
   "version": 1,
-  "enabled-engines": ["vertx","postgres","kafka","flink"],
+  "enabled-engines": ["vertx", "postgres", "kafka", "flink"],
   "engines": {
     "flink": {
-      "connectors": {
-        "postgres": { "connector": "jdbc-sqrl", … },
-        "kafka":   { "connector": "kafka", … },
-        "kafka-keyed": { … },
-        "kafka-upsert": { … },
-        "localfile": { … },
-        "iceberg": { … },
-        "print": { "connector": "print" }
+      "config": {
+        "execution.runtime-mode": "STREAMING",
+        "rest.address": "localhost",
+        "rest.port": 8081,
+        "state.backend.type": "rocksdb",
+        ...
       }
     },
     "snowflake": {
       "schema-type": "aws-glue",
       "catalog-name": "${SNOWFLAKE_CATALOG_NAME}",
       "external-volume": "${SNOWFLAKE_EXTERNAL_VOLUME}",
-      "url": "jdbc:snowflake://${SNOWFLAKE_ID}.snowflakecomputing.com/?…"
+      "url": "jdbc:snowflake://${SNOWFLAKE_ID}.snowflakecomputing.com/?..."
     }
   },
+  "connectors": {
+    "postgres": { "connector": "jdbc-sqrl", ... },
+    "kafka":   { "connector": "kafka", ... },
+    "kafka-keyed": { ... },
+    "kafka-upsert": { ... },
+    "localfile": { ... },
+    "iceberg": { ... },
+    "print": { "connector": "print" }
+  }
   "test-runner": {
+    "snapshot-dir": "./snapshots",
+    "test-dir": "./tests",
     "delay-sec": 30,
-    "mutation-delay": 1,
-    "headers": {
```

Comment:
what happened to headers?

Reply:
Nothing, but those are not part of the default configuration, so I removed it from that section.